next.config.js

```diff
@@ -1,8 +1,18 @@
 const withSass = require('@zeit/next-sass');
+const withCSS = require('@zeit/next-css');
 
-let nextConfig = withSass();
-
-// You can pass nextConfig to another plugin, for example:
-//nextConfig = withMdx(nextConfig)
-
-module.exports = nextConfig;
\ No newline at end of file
+module.exports = withCSS(withSass({
```

Comment:
Adding support for importing css

Example: `import 'leaflet/dist/leaflet.css';`